cmd/wallet-web/src/translations/en.js

```diff
@@ -4,6 +4,7 @@ import DashboardLayout from './DashboardLayout/en.json';
 import Signin from './Signin/en.json';
 import Signout from './Signout/en.json';
 import Signup from './Signup/en.json';
+import Credentials from './Credentials/en.json';
```

Comment:
translation file name is too generic, how about credential view, share, save? something to revisit later.

Reply:
@talwinder50 ^